OpenOversight/tests/test_csvs/.gitignore

```diff
@@ -0,0 +1 @@
+!*.csv
```

Comment:
Why was this `.gitignore` necessary?  (I think I might be missing something here.)

Reply:
we have a `.gitignore` on the repository root level that includes `.csv` files which makes some sense, since we don't want to bring real world data into this repository. So I added this `.gitignore` file to basically revert the ignore on csv files in that particular directory so I can add the `.csv` files used in some of the tests.